.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Include, if possible, a sample file that exhibits the behavior
+2. Include minimal but relevant ruby-marc code that exhibits the behavor
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Program Output**
+If applicable, add program output and/or backtraces
+
+**Environment (please complete the following information):**
+ - ruby-marc version
+ - ruby runtime and version (e.g. MRI 2.7, JRuby 9.2.11.0)
+ - OS
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```

Comment:
I'm thinking maybe we don't need this :-)

Reply:
Removing, even though my phone has more cores than my laptop does.